src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { MuiAlert } from '../Alert';
+import { Typography } from '../Typography';
+
+import ErrorS from '../../icons/banner/error-s.svg';
+import ErrorXS from '../../icons/banner/error-xs.svg';
+import InfoS from '../../icons/banner/info-s.svg';
+import InfoXS from '../../icons/banner/info-xs.svg';
+import NeutralS from '../../icons/banner/neutral-s.svg';
+import NeutralXS from '../../icons/banner/neutral-xs.svg';
+import SuccessS from '../../icons/banner/success-s.svg';
+import SuccessXS from '../../icons/banner/success-xs.svg';
+import WarningS from '../../icons/banner/warning-s.svg';
+import WarningXS from '../../icons/banner/warning-xs.svg';
+
+type IconSize = 's' | 'xs';
```

Comment:
This would be more a question for design, but if we don't have more sizes why don't we just call them `default` & 's' or something like thts

Reply:
would prefer to call them `xs` and `s` but have, let's say, `s` as a default